src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java

```diff
@@ -36,31 +38,38 @@
 
 @Component
 public class FieldValidationAction {
-
+    private final Logger logger = LoggerFactory.getLogger(FieldValidationAction.class);
```

Comment:
Added debug logging to diagnose the issues with the jobs.  If we have a problem with configurations being invalid we can turn on debug logging and get more information.